Trigger/data/cprDeM_highP_stopTarg/main_cprDeM_highP_stopTarg_hsFilter.fcl

```diff
@@ -3,8 +3,8 @@ physics.filters.cprDeMHighPStopTargHSFilter.doHelicityCheck  : true
 physics.filters.cprDeMHighPStopTargHSFilter.helicity      : 1
 physics.filters.cprDeMHighPStopTargHSFilter.minNStrawHits : 15.0
 physics.filters.cprDeMHighPStopTargHSFilter.minHitRatio   : 0.4
-physics.filters.cprDeMHighPStopTargHSFilter.minMomentum   : 80.0
-physics.filters.cprDeMHighPStopTargHSFilter.maxMomentum   : 140.0
+physics.filters.cprDeMHighPStopTargHSFilter.minMomentum   : 70.0
```

Comment:
Should there be a prolog 'minMomentum' that can be used by both cpr and tpr?

Reply:
In the OTS implementation they share the same parameter. Here in the Offline this is just a test stand while we are waiting for the real DB to be up and running. These tables are not touched frequently, so I think that we can wait for the DB implementation and avoid complicating the system with another prolog file.